### ISSUE TITLE
GNC inside EXG in parallel

### ISSUE THREAD

**Issue by tandreasr** [MODFLOW-USGS/modflow6#1733]
Hi,
after corresponding with mjr-deltares I finally got 2 new issues regarding parallel execution.
I've attached a test model.
When I run it (Windows, latest nightly build) with

"...\mf6.5.0.dev2_win64par\bin\mpiexec.exe" -np 9 "...\mf6.5.0.dev2_win64par\bin\mf6.exe" --PARALLEL mfsim.nam

it crashes with:

ERROR REPORT:
  1. Cell2d number in list (1018) is outside of the grid. Cell number cannot
     be determined for cellid (1 1018) and results in a user node number (-1)
     that is outside of the grid.

This is a valid cell and the problem runs properly with -np 1.
Obviously the parallel mode can't handle GNC files in exchanges.
Commenting the GNC entry inside GWF1_1-GWF1_2.RIGHT.EXG lets the model run without errors.

For further testing there are similiar commented entries in most other EXG files.

The GNC files inside the NAM files seem to work fine
[__GNC Issue.zip](https://github.com/MODFLOW-USGS/modflow6/files/14998343/__GNC.Issue.zip)

**Comment by mjr-deltares** [comment]
The problem was not trapping on GNC enabled in parallel, since it is not supported. The check is included in PR #1738.

**Comment by mjr-deltares** [comment]
Alternatively, you should be able to set XT3D active on just the interface to avoid large effects from the grid refinement. I do notice an issue with enabling that on your model, but I will have to look a bit further. Will get back on that one.

**Comment by mjr-deltares** [comment]
I have created a new issue for the issue with XT3D so we can close this one if the fix goes in
